### PR TITLE
feat: Added modern-lilac and modern-lilac2 themes

### PR DESCRIPTION
**Pull request by FleetAdmiralJakob**
## Description

I added the best theme ever!!!!!
Actually it's my personal theme, but I use it all the time and I think it belongs here too.

### Type of change

- [x] New feature (added a non-breaking change which adds functionality)

## How Has This Been Tested?

With the attribute thing in the link

## Checklist:

- [x] I have checked to make sure no other [pull requests](https://github.com/DenverCoder1/github-readme-streak-stats/pulls?q=is%3Apr+sort%3Aupdated-desc+) are open for this issue
- [x] The code is properly formatted and is consistent with the existing code style

### CODE REVIEW

File: src/themes.php [hunk 1]
```diff
@@ -1082,4 +1082,16 @@
...
+        "border" => "#1C1E26",
+        "stroke" => "#1C1E26",
+        "ring" => "#5d417a",
+        "fire" => "#5d417a",
+        "currStreakNum" => "#FAB795",
+        "sideNums" => "#c770f0",
+        "currStreakLabel" => "#FAB795",
+        "sideLabels" => "#c770f0",
+        "dates" => "#FAB795",
+    ]
```
Comment (DenverCoder1) [inline src/themes.php:1096]: For consistency, there should be a trailing comma on this line.
```suggestion
    ],
```

**Review by DenverCoder1 — COMMENTED** [review]
Looks great!

It would also be nice if you could update the [`docs/themes.md`](https://github.com/FleetAdmiralJakob/github-readme-streak-stats/blob/main/docs/themes.md) as well on your branch.

image

https://user-images.githubusercontent.com/20955511/197569406-6ff144c3-1d6e-4500-9f0b-3112a6c62584.png

Here is an image URL you can use.

**Comment by FleetAdmiralJakob** [comment]
Hi, I thought it would be nice if there is a second theme where the text is white, so I made it und modern-lilac2. If you send me again such an image I will add this theme also to the doc so that you can merge this PR.

**Comment by DenverCoder1** [comment]
The image is from the demo site

> image

`https://user-images.githubusercontent.com/20955511/197575977-029fc730-9c7e-4556-be7c-a727a1715fa7.png`

**Comment by FleetAdmiralJakob** [comment]
Did it, you can merge!!!!

**Review by DenverCoder1 — APPROVED** [review]
Looks great! Thanks for the contribution :tada:

**Pull request merged**